aws-creds/iam.tf

```diff
@@ -93,6 +98,10 @@ resource "aws_iam_role" "terraform_architect_iam_role" {
   name = "${var.iam_prefix}terraform-architect"
   path = "/"
   assume_role_policy = data.aws_iam_policy_document.terraform_user_assume_policy.json
+  tags = {
+    "Terraform" = "True"
+  }
+  permissions_boundary = "arn:aws:iam::328656936502:policy/TestRoleCreationBoundary"
```

Comment:
I think this doesn't belong here either?

Reply:
we addressed this in slack, I commented it out